src/main/java/org/folio/edge/core/utils/OkapiClient.java

```diff
@@ -152,26 +163,30 @@ public void post(String url, String tenant, String payload, MultiMap headers,
           request.headers().get(X_OKAPI_TOKEN)));
     }
 
-    request.handler(responseHandler)
-      .exceptionHandler(exceptionHandler)
-      .setTimeout(reqTimeout);
+    // TODO   .setTimeout(reqTimeout);
```

Comment:
setConnectTimeout?

Reply:
Forgot to remove that comment.  I have added setConnectTimeout (already added setIdleTimeout).

Reply:
You fixed it, thanks, can be closed.